service/init.go

```diff
@@ -0,0 +1,45 @@
+package service
+
+import (
+	"os"
+	"tat_gogogo/infrastructure/router"
+	"tat_gogogo/utilities/logs"
+
+	"github.com/gin-contrib/cors"
+	"github.com/gin-contrib/pprof"
+	"github.com/gin-gonic/gin"
+	"github.com/spf13/viper"
+)
+
+func configInit() {
+	viper.SetConfigName("config")
+	viper.AddConfigPath(".")
+	err := viper.ReadInConfig()
+	if err != nil {
+		panic("Fatal error config file: " + err.Error())
+	}
+}
+
+func ginInit() *gin.Engine {
+	ginRouter := gin.Default()
+	// -----------------------------------------------[Middleware]
+	// ------------------------------------------[CORS]
```

Comment:
could the comment line `-----` be the same length?

Reply:
but we could discuss the comment style, the empty line we are not going to mark `//--------------` 
and use the c++ style.
ref: https://golang.org/doc/effective_go.html#commentary

Reply:
I remeber all is high same length.
because i want split event function and song find the.